sdk/cosmos/tests.yml

```diff
@@ -211,13 +211,37 @@ jobs:
           ProfileFlag: '-Pe2e'
           AdditionalArgs: '-DargLine="-Dcosmos.directModeProtocol=Https"'
           DESIRED_CONSISTENCY: 'Strong'
+        Bounded_Staleness_Https_E2E:
+          DisplayName: Bounded Staleness Single Region ReadMyWrites
+          OSVmImage: 'windows-2019'
+          ProfileFlag: '-Pe2e'
+          AdditionalArgs: '-DargLine="-Dcosmos.directModeProtocol=Https"'
+          DESIRED_CONSISTENCY: 'BoundedStaleness'
+        Consistent_Prefix_Https_E2E:
+          DisplayName: Consistent Prefix Single Region ReadMyWrites
+          OSVmImage: 'windows-2019'
```

Comment:
please use Linux as the OS image. here and elsewhere in the new bounded-staleness and consistent prefix tests

Reply:
Do we need to change the existing ReadMyWrites as well to ubuntu machines ?
Or just Bounded Staleness and Consistent Prefix ??

Reply:
they are using windows too? hmm. ok. out of scope of this PR really. I retract this comment we can come back to it later if needed outside of scope of this PR.

Reply:
NO op for this PR as this is outside of scope of this PR.